apollo-runtime/src/main/java/com/apollographql/apollo/ApolloCall.java

```diff
@@ -13,25 +13,83 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+/**
+ * ApolloCall is an abstraction for a request that has been prepared for execution. ApolloCall
+ * represents a single request/response pair and cannot be executed twice, though it can be cancelled.
```

Comment:
I guess it worth to mention smth like to execute it twice `clone` should be used